File: Python PowerPoint Examples/ShapesApiDemo/deleteSlideShape.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudpowerpoint.api.shapes_api.ShapesApi(configuration)

fileName = "DeleteSlideShape.pptx"
password = ""
storage = ""
inputFolder = "input"
slide_index = 0
shape_index = "0"
result =api.delete_slide_shape(fileName, slide_index, shape_index,password=password, storage=storage,folder=inputFolder)

File: PHP PowerPoint Examples/ShapesApiDemo/deleteSlideShape.php
<?php

use Spire\Cloud\Powerpoint\Sdk\Api\ShapesApi;
use Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new ShapesApi($configuration);

$name = "DeleteSlideShape.pptx";
$password = null;
$folder = "input";
$storage = null;

$slideIndex = 0;
$shapeIndex = "0";
$apiInstance->deleteSlideShape($name, $slideIndex, $shapeIndex, $password, $folder, $storage);